en/docs/consume/test-api.md

```diff
@@ -4,18 +4,19 @@ Before using your API in production, you can test it using the integrated consol
 
 This guide will take you through the steps to test an API in Choreo. 
 
-1. Sign in to Choreo Developer Portal at [https://devportal.choreo.dev](https://devportal.choreo.dev). Alternatively, click **Developer Portal**  on the Choreo console header. 
+1. Sign in to Choreo Developer Portal at [https://devportal.choreo.dev](https://devportal.choreo.dev). Alternatively, click **Developer Portal**  on the Choreo Console header. 
 
     ![Developer Portal](../assets/img/developer-portal/manage-applications/developer-portal.png){.cInlineImage-half}
 
 2. Click **APIs**.
+
 3. Select an API you want to test and click **Try Out** on the card. In this case, skip step 4. Alternatively, you can click on the API and follow step 4. 
 4. In the left pane, click **Try Out**.
 5. Select the required environment from the **Environment** list.
 
     ![Test Panel](../assets/img/developer-portal/test-api/test-panel.png){.cInlineImage-half}
 
-6. From the **Subscribed Application** list, select the relevant application you subscribed the API to. To learn how to subscribe to an application, see [Manage Subscriptions](../manage-subscription/). 
+6. From the **Subscribed Application** list, select the application you used to subsribe the API to. To learn how to subscribe to an application, see [Manage Subscriptions](../manage-subscription/). 
```

Comment:
"subsribe" should change to "subscribe".

Reply:
Suggestion:
"From the **Subscribed Application** list, select the application via which you subscribed to the API. To learn how an application can subscribe to an API, see [Manage Subscriptions](../manage-subscription/)."